### PR TITLE
Add `format` qualifier to `fl_waveform`

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
Add `Format` qualifier to `fl_waveform`

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [x] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: url-gen/src/main/kotlin/com/cloudinary/transformation/effect/EffectActions.kt [hunk 1]
```diff
@@ -488,9 +488,10 @@ class MakeTransparent private constructor(private val tolerance: Any?, private v
     }
 }
 
-class Waveform private constructor(private var color: Any?, private val backgroundColor: Color?) : Effect() {
+class Waveform private constructor(private var color: Any?, private val backgroundColor: Color?, private val format: Format?) : Effect() {
```
Comment (const-cloudinary) [inline url-gen/src/main/kotlin/com/cloudinary/transformation/effect/EffectActions.kt:491]: Format is a mandatory parameter

File: url-gen/src/test/kotlin/com/cloudinary/transformation/EffectTest.kt [hunk 2]
```diff
@@ -156,7 +156,8 @@ class EffectTest {
         cldAssert("fl_waveform", Effect.waveform())
 
         cldAssert(
-            "b_white,co_black,fl_waveform", Effect.waveform {
+            "b_white,co_black,f_png,fl_waveform", Effect.waveform {
```
Comment (const-cloudinary) [inline url-gen/src/test/kotlin/com/cloudinary/transformation/EffectTest.kt:159]: According to spec, it should be `VideoEdit.waveform` and not `Effect.waveform`

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: url-gen/src/main/kotlin/com/cloudinary/transformation/effect/EffectActions.kt (hunk 1 above)
Comment (adimiz1) [reply url-gen/src/main/kotlin/com/cloudinary/transformation/effect/EffectActions.kt:491]: Thank you, missed that, fixing

File: url-gen/src/test/kotlin/com/cloudinary/transformation/EffectTest.kt (hunk 2 above)
Comment (adimiz1) [reply url-gen/src/test/kotlin/com/cloudinary/transformation/EffectTest.kt:159]: Checking, thank you

**Review by const-cloudinary — APPROVED** [review]

**Pull request merged**